x-pack/legacy/plugins/snapshot_restore/server/routes/api/policy.test.ts

```diff
@@ -360,4 +361,25 @@ describe('[Snapshot and Restore API Routes] Policy', () => {
       ).rejects.toThrow();
     });
   });
+
+  describe('executeRetentionHandler()', () => {
```

Comment:
Not entirely sure I understand what coverage this test provides?

Reply:
good point, i remove this test for now.